src/Console/MakeCommand.php

```diff
@@ -67,4 +68,18 @@ protected function getOptions(): array
             ['force', 'f', InputOption::VALUE_NONE, 'Create the Folio page even if the page already exists'],
         ];
     }
+
+    /**
+     * Prompt for missing input arguments using the returned questions.
+     *
+     * @return array
+     */
+    protected function promptForMissingArgumentsUsing()
+    {
+        return [
+            'name' => class_exists(Application::class) && version_compare(Application::VERSION, '10.17.0', '>=')
+                ? ['What should the page be named?', 'E.g. users/index, users/[User]']
+                : 'What should the page be named?',
+        ];
```

Comment:
The Laravel version check exists purely to support Laravel 10.15 and 10.16, which are supported by Folio, but do not support a tuple here. We could alternatively bump the minimum requirements of this package.

The `class_exists` is because `illuminate/foundation` is not a dependency of this project so phpstan wasn't happy. I'm not aware of any use case for Folio outside of a Laravel application.